vlib/v/checker/tests/infix_dup_in_err.vv

```diff
@@ -0,0 +1,6 @@
+if 1 in [1, 2, 2, 2] { println('ok') }
+
+a := `a`
+if `a` in [`c`, a, a] { println('ok') }
```

Comment:
What about 

``` 
 a := `a` 
 b := `a` // same value of 'a' variable to ensure that checker is for variable name and not variable value
 if `a` in [`c`, b, a, a] { println('ok') } 

```

Reply:
and with the constants too

Reply:
Yes

Reply:
It just does static analysis, not dynamic analysis. Because we could have more complex conditionals to handle (as ternary, function call, etc).

Reply:
The simple static analysis that it does currently, is fine to cover most cases where `x in [...]` is used.